lib/semantics/check-do-concurrent.cc

```diff
@@ -160,25 +192,38 @@ class DoConcurrentEnforcement {
     }
     return false;
   }
+  bool addName(const std::optional<parser::Name> &nm) {
+    if (nm.has_value())
```

Comment:
Please use braces on all structured statements for consistency with the rest of F18.  `clang-format` might be able to add them for you.

It would be more clear if this function returned `void` and its callers returned explicit `true`, rather than forwarding a `true` result back.  As written, these `Pre` routines look as if their return values that guide the parse tree traversal might be determined by the prior presence of the names in the set.

Reply:
Thanks @klausler. I tried using clang-format, but that showed some differences with the existing code. Just wanted to check whether you use clang-format with any settings or is it a plain run of clang-format?

Reply:
Please ignore my comment. I now find that running without any style options shows no differences.

Reply:
There's a `.clang-format` file in the top-level directory that contains the settings that we use.

Reply:
Added braces. Changed addName function to return void and its callers now return true.